lib/kubernetes-deploy/kubernetes_resource/thirdpartyresource.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class ThirdPartyResource < GenericResource
```

Comment:
This class allows `k8s-deploy` to monitor deployment of third party resource *definitions*.